data_steward/deid/parser.py

```diff
@@ -107,6 +107,17 @@ def query_priority(priority):
         raise ArgumentTypeError(message)
 
 
+def odataset_name_verification(odataset_name):
+    """
+    Ensures the output dataset name contains _deid
+    """
```

Comment:
documentation strings should follow the format.
```
single line summary
BLANK LINE
paragraph style explanation if needed
BLANK LINE
:param PARAMETER NAME:  parameter description
:return:  return type description
```
For more info, https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings